packages/probers/cypress/e2e/favoriteTrack.cy.ts

```diff
@@ -12,7 +12,9 @@ describe('Favorite Track', () => {
     const base64Entropy = Buffer.from(user.entropy).toString('base64')
     cy.visit(`${track.route}?login=${base64Entropy}`)
 
-    cy.findByText(user.name, { timeout: Cypress.env('initialLoadTimeout') }).should('exist')
+    cy.findByText(user.name, {
+      timeout: Cypress.env('initialLoadTimeout')
```

Comment:
Sup here?

Reply:
wtf I didn't change this or the other tests... 😵 reverting those changes